paf-mvp-demo-express/scripts/generate-examples.ts

```diff
@@ -33,17 +33,29 @@ import {OperatorApi} from "@operator/operator-api";
 import {GetNewIdResponseBuilder, GetIdsPrefsResponseBuilder, PostIdsPrefsResponseBuilder, Get3PCResponseBuilder, GetIdentityResponseBuilder} from "@core/model/response-builders";
 import {Validator} from "jsonschema";
 import {publicKeys} from "../src/public-keys";
-
-const fs = require('fs').promises;
+import * as fs from "fs";
 
 const getTimestamp = (dateString: string) => getTimeStampInSec(new Date(dateString))
 const getUrl = (method: "POST"|"GET", url: URL): string => `${method} ${url.pathname}${url.search}\nHost: ${url.host}`
 const getGetUrl = (url: URL): string => getUrl("GET", url)
 const getPOSTUrl = (url: URL): string => getUrl("POST", url)
 const getRedirect = (url: URL): string => `303 ${url}`
 
+if (!(process.argv[2]?.length > 0)) {
+    throw `Usage: ts-node -r tsconfig-paths/register generate-examples.ts <outputDir>\nExample: ts-node -r tsconfig-paths/register generate-examples.ts ../../../addressable-network-proposals/mvp-spec/partials`
```

Comment:
Instead of a string on one line with `\n`, you can do a multi-line string with simple quotes in typescript. It will be more readable.

Reply:
Good point, and you gave me inspiration to improve this use case ;-)
thanks